desktop/libs/notebook/src/notebook/connectors/hiveserver2.py

```diff
@@ -189,7 +190,7 @@ def create_session(self, lang='hive', properties=None):
       session = None
 
     reuse_session = session is not None
-    if not reuse_session:
+    if not reuse_session or HPLSQL.get():
```

Comment:
Could me move this to the previous if to keep the logic in the same place?

Isn't re-creating the session all the time going to be an issue?

Reply:
Yes, creating a new one but,
if I am not using new sessions every time, it is going to use the old one, and that session might be of non hplsql mode.
Not found any way to separate the session?

Reply:
Have you looked at the application var just above?

Why would it be an issue in DWX?

Reply:
Have you looked at the application var just above?

Why would it be an issue in DWX?